mws/utils.py

```diff
@@ -271,22 +272,6 @@ def unique_list_order_preserved(seq):
     return [x for x in seq if not (x in seen or seen_add(x))]
 
 
-def dt_iso_or_none(dt_obj):
```

Comment:
Removing this method is a fair change. I added it myself from a version of the API I was using in production years ago, but under current circumstances it doesn't serve much of a purpose.

While this method did provide some enforcement requiring a user to pass a `datetime` to a request method (and get an early exception if they didn't), I think that's an acceptable risk for the time being. We could also offer type hints ala [PEP 484](https://www.python.org/dev/peps/pep-0484/), but that is a topic for another day.